apps/condo/domains/miniapp/gql.js

```diff
@@ -18,9 +18,9 @@ const ALL_MINI_APPS_QUERY = gql`
     }
 `
 
-const SEND_APP_PUSH_MESSAGE_MUTATION = gql`
-    mutation sendAppPushMessage ($data: SendAppPushMessageInput!) {
-        result: sendAppPushMessage(data: $data) { id status }
+const SEND_B2C_APP_PUSH_MESSAGE_MUTATION = gql`
+    mutation sendB2CAppPushMessage ($data: SendB2CAppPushMessageInput!) {
```

Comment:
Why is this renaming necessary? Could anyone use this mutation?

Reply:
There will be other service for B2B messages when required. This renaming was discussed and approved with @pahaz  :)